assessments/PALS/seeds/assessmentCategoryDescriptors.csv

```diff
@@ -0,0 +1,2 @@
+codeValue,description,namespace,shortDescription
+Readiness,Assessment designed to measure readiness for a particular grade or subject.,uri://ed.sc.gov/PALS/AssessmentCategoryDescriptor,Readiness
```

Comment:
it appears that this assessment is no SC specific, so it's namespace should not be ed.sc.gov. It appears that it was owned by illuminate, which was bought by renaissance. I would use `renaissance.com/products`